signalfx-async-wrapper.js

```diff
@@ -12,67 +14,87 @@ class SignalFxWrapper {
     originalContext,
     originalCallback,
     dimensions,
-    accessToken
+    disableTracing,
+    disableMetrics
   ) {
     this.originalObj = originalObj;
     this.originalFn = originalFn;
     this.originalEvent = originalEvent;
     this.originalContext = originalContext;
     this.originalCallback = originalCallback;
+    this.enableMetrics = sfxHelper.isMetricsEnabled(disableMetrics);
+    this.enableTracing = sfxHelper.isTracingEnabled(disableTracing);
```

Comment:
nit, args are negated by helper and the enable properties are then negated in all checks?  I think consistent disable/enable would be clearer.